quests.pl

```diff
@@ -0,0 +1,40 @@
+:- dynamic not_discovered_quest/1, current_quest/1, finished_quest/1.
+
+quest(killing_monsters).
+quest(spiderman).
+quest(bounty_hunter).
+quest_description(killing_monsters, "There is a mighty dragon in the west, kill him.").
+quest_description(spiderman, "Old grandma is scared of the spider - get rid of it.").
+quest_description(bounty_hunter, "Grandma gave you a key to treasure, find it and get your reward.").
+not_discovered_quest(killing_monsters).
```

Comment:
I was wondering about showing use what he hasn't done, but for know we can remove not_discovered_quest.